src/acceptance-test/java/org/zalando/nakadi/repository/db/TimelineDbRepositoryTest.java

```diff
@@ -0,0 +1,148 @@
+        modified.setLatestPosition(pos);
+
+        tRepository.updateTimelime(modified);
+        final Timeline result = tRepository.getTimeline(modified.getId()).get();
+
+        Assert.assertEquals(modified, result);
+        Assert.assertNotEquals(initial, result);
+    }
+
+    @Test(expected = DuplicateKeyException.class)
+    public void testDuplicateOrderNotAllowed() throws InternalNakadiException, DuplicatedEventTypeNameException {
+        final Storage storage = sRepository.createStorage(
+                StorageDbRepositoryTest.createStorage("default", "test", "path"));
+        final EventType testEt = eRepository.saveEventType(TestUtils.buildDefaultEventType());
+
+        tRepository.createTimeline(createTimeline(
+                storage, UUID.randomUUID(), 0, "test_topic", testEt.getName(),
+                new Date(), null, null, null));
+        tRepository.createTimeline(createTimeline(
+                storage, UUID.randomUUID(), 0, "test_topic", testEt.getName(),
+                new Date(), null, null, null));
+    }
+
+    @Test
+    public void testListTimelinesOrdered() throws InternalNakadiException, DuplicatedEventTypeNameException {
+        final Storage storage = sRepository.createStorage(
+                StorageDbRepositoryTest.createStorage("default", "test", "path"));
+        final EventType testEt = eRepository.saveEventType(TestUtils.buildDefaultEventType());
+        final Timeline t1 = tRepository.createTimeline(createTimeline(
+                storage, UUID.randomUUID(), 1, "test_topic", testEt.getName(),
+                new Date(), null, null, null));
+        final Timeline t0 = tRepository.createTimeline(createTimeline(
+                storage, UUID.randomUUID(), 0, "test_topic", testEt.getName(),
+                new Date(), null, null, null));
+
+        final List<Timeline> testTimelines = tRepository.listTimelines(testEt.getName());
+        Assert.assertEquals(2, testTimelines.size());
+        Assert.assertEquals(t0, testTimelines.get(0));
+        Assert.assertEquals(t1, testTimelines.get(1));
+    }
+
+    @Test
+    public void testTimelineDeleted() throws InternalNakadiException, DuplicatedEventTypeNameException {
+        final Storage storage = sRepository.createStorage(
+                StorageDbRepositoryTest.createStorage("default", "test", "path"));
+        final EventType testEt = eRepository.saveEventType(TestUtils.buildDefaultEventType());
+        final Timeline t1 = tRepository.createTimeline(createTimeline(
+                storage, UUID.randomUUID(), 1, "test_topic", testEt.getName(),
+                new Date(), null, null, null));
+        Assert.assertEquals(1, tRepository.listTimelines(testEt.getName()).size());
+        tRepository.deleteTimeline(t1.getId());
+        Assert.assertEquals(0, tRepository.listTimelines(testEt.getName()).size());
+    }
+
+    private static Timeline createTimeline(
```

Comment:
I don't think that using builder in this case is necessary. I will create static method for timeline creation as a part of `@Nullable` policy.